pegasus/sites.v3/code.org/layouts/wide.haml

```diff
@@ -1,9 +1,9 @@
 .wrapper
   -if header['theme'] == 'responsive_wide'
     #desktop-header.desktop-feature
-      -#= view :header
+      -# = view :header
```

Comment:
@breville any reason to keep this here or can I send you a PR with the reverse logic?